app/kubernetes/views/deploy/deployController.js

```diff
@@ -15,11 +15,35 @@ class KubernetesDeployController {
     this.KubernetesResourcePoolService = KubernetesResourcePoolService;
     this.StackService = StackService;
 
+    this.deployOptions = [
+      { id: 'method_kubernetes', icon: 'fa fa-cubes', label: 'Kubernetes', description: 'Kubernetes manifest format', value: KubernetesDeployManifestTypes.KUBERNETES },
+      { id: 'method_compose', icon: 'fa fa-docker', label: 'Compose', description: 'docker-compose format', value: KubernetesDeployManifestTypes.COMPOSE },
+    ];
+
+    this.methodOptions = [
+      { id: 'method_repo', icon: 'fab fa-github', label: 'Git Repository', description: 'Use a git repository', value: KubernetesDeployBuildMethods.GIT },
+      { id: 'method_editor', icon: 'fa fa-edit', label: 'Web editor', description: 'Use our Web editor', value: KubernetesDeployBuildMethods.WEB_EDITOR },
+    ];
```

Comment:
Not specially needed in this PR but as we are usign the box selector in many other places, can we extract the option object as a class (or func) so we can have a clear overview of expected fields?

Smth like
```js
class Option {
  constructor({id, icon, label, description, value}) {
    Object.assign(this, {id, icon, label, description, value});
  }
}

///

this.deployOptions = [
  new Option({
    id: 'method_kubernetes',
    icon: 'fa fa-cubes',
    label: 'Kubernetes',
    description: 'Kubernetes manifest format',
    value: KubernetesDeployManifestTypes.KUBERNETES
  }),
  new Option({ id: 'method_compose', icon: 'fa fa-docker', label: 'Compose', description: 'docker-compose format', value: KubernetesDeployManifestTypes.COMPOSE }),
]
```

it will make things easier for future usage.

Reply:
hmm, sure? but I prefer just a function that receives the parameters and returns the object.

Reply:
Func is good aswell as long as we have a reference somewhere to know the fields the box selector item is expecting